searx/engines/startpage.py

```diff
@@ -46,24 +51,23 @@
 # do search-request
 def request(query, params):
 
-    params['url'] = search_url
-    params['method'] = 'POST'
-    params['data'] = {
+    args = {
         'query': query,
         'page': params['pageno'],
         'cat': 'web',
-        'cmd': 'process_search',
-        'engine0': 'v1all',
+        # 'abp': "-1",
+        'sc': 'Mj4jZy61QETj20',
```

Comment:
This `sc` argument is new.  I copied the value from an interactive startpage session.  I don't know what is encoded in the string.  This string might be invalid at one day.

Reply:
Yeah, I think we should test this string again in 24 hours.

Reply:
in case the `sc` parameter needs to be fetched: gigablast.py fetches the extra parameters every hour
https://github.com/searxng/searxng/blob/00d3a7477c01981a6ef2d396332670ad5b831fa6/searx/engines/gigablast.py#L81-L84

Reply:
I installed the patch on my server / lets see if this link will work in 24 https://darmarit.org/searx/search?q=%21sp%20Salvadore%20dali

Reply:
FYI `sc` is no longer valid / I will look for a solution like @dalf suggested in https://github.com/searxng/searxng/pull/695#discussion_r778790532

Reply:
To get an actual `sc` time stamp from startpage's home page I pushed b3ea061 .. from my local IP it works, but I did not have luck when I tested this patch on my server .. I have to dig a little bit deeper :-(

Reply:
Maybe they already put your server IP on some kind of blacklist so that it doesn't get new `sc` values?

Reply:
This is what I also assume .. I will wait a day and see if it works, if not I have to analyze what the reason is.

Reply:
> To get an actual `sc` time stamp from startpage's home page I pushed [b3ea061](https://github.com/searxng/searxng/commit/b3ea061a608af854614da067f83789f0001c68ff) .. from my local IP it works, but I did not have luck when I tested this patch on my server .. I have to dig a little bit deeper :-(

@return42 I just tried your patch on my server and it seems to work !

Reply:
@Stratos42 thanks for your feedback ... I did not have luck on my server, may be IP is blocked since a longer time .. may be something other went wrong .. before I merge this PR I will invest in more research .. 

Please keep me informed if the state of your server change / thanks!